### PR TITLE
Add new core committers

### PR DESCRIPTION
**Pull request by jwilander**

### CODE REVIEW

**Comment by jwilander** [comment]
@hanzei @DSchalla @cometkim let me know if those dev areas look right or if we should add/remove any

File: site/content/contribute/getting-started/core-committers.md [hunk 1]
```diff
@@ -49,6 +49,15 @@ Below is the list of core committers working on Mattermost:
...
   - Dev areas: Web App, React, Redux, Performance
+- **Daniel Schalla**
+  - @dschalla on [pre-release.mattermost.com](https://pre-release.mattermost.com/core/messages/@dschalla) and [@dschalla](https://github.com/dschalla) on GitHub
+  - Dev areas: Security, Plugins, Integrations, Performance
+- **Hyeseong Kim**
+  - @cometkim on [pre-release.mattermost.com](https://pre-release.mattermost.com/core/messages/@cometkim) and [@cometkim](https://github.com/cometkim) on GitHub
+  - Dev areas: Web App, React, Redux, REST API
+- **Hanzei**
+  - @hanzei on [pre-release.mattermost.com](https://pre-release.mattermost.com/core/messages/@hanzei) and [@hanzei](https://github.com/hanzei) on GitHub
+  - Dev areas: Plugins, Integrations, Matterpoll
```
Comment (hanzei) [inline site/content/contribute/getting-started/core-committers.md]: Please remove `Matterpoll`. Its a personal project and not an "official" Mattermost project.

Would you please add `Security`? I know I did't a contribution related to this, but its a topic which interests me.

Comment (jwilander) [reply site/content/contribute/getting-started/core-committers.md]: For sure, I'll update it

**Comment by jasonblais** [comment]
(Sorry, I should have left the PR open and not merge it right away)

File: site/content/contribute/getting-started/core-committers.md (hunk 1 above)
Comment (jwilander) [reply site/content/contribute/getting-started/core-committers.md]: https://github.com/mattermost/mattermost-developer-documentation/commit/da9974ad7d203da0bdf0b0541eb430b9746d6426

**Comment by jwilander** [comment]
No worries, I can commit directly to master for any updates

**Comment by DSchalla** [comment]
Works for me, thanks!

**Pull request merged**